common/component/kafka/metadata.go

```diff
@@ -80,16 +89,23 @@ type KafkaMetadata struct {
 	Version                string              `mapstructure:"version"`
 	internalVersion        sarama.KafkaVersion `mapstructure:"-"`
 	internalOidcExtensions map[string]string   `mapstructure:"-"`
+
+	// configs for kafka client
+	clientConnectionRefreshInterval   time.Duration `mapstructure:"clientConnectionRefreshInterval"`
```

Comment:
should this fields be public? `ClientConnectionRefreshInterval` and `ClientConnectionKeepAliveInterval`